cardinal/utils/decimal.go

```diff
@@ -1,69 +0,0 @@
-package utils
```

Comment:
looks good but didn't scott mention something about him creating this decimal library and he wanted someone to review it? are we upstreaming this to cardinal or just getting rid of it completely? what was it used for before?

Reply:
yeah we should upstream something like this to cardinal. we need deterministic floating point lib